.github/workflows/maven.yml

```diff
@@ -0,0 +1,22 @@
+# This workflow will build a Java project with Maven
+# For more information see: https://help.github.com/actions/language-and-framework-guides/building-and-testing-java-with-maven
+
+name: Java CI with Maven
+
+on: [push]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up JDK 1.8
+      uses: actions/setup-java@v1
+      with:
+        java-version: 1.8
+    - name: Build with Maven
+      run: mvn -B package --file pom.xml -DskipTests
```

Comment:
Why are we skipping the tests? Shouldn't we block a PR if it make the tests fail?

Reply:
some of the tests work only from Google Compute Engine because of some authorization things, also they require sensitive data and I'm not sure that github can somehow access it without adding this data to repo

Reply:
Fair enough. For now, let's just agree on running these tests locally before merging the PRs then.

Reply:
Sure!